deploy-service/common/src/main/java/com/pinterest/deployservice/db/QueryUtils.java

```diff
@@ -30,6 +32,10 @@ public static String genStringGroupClause(Collection<String> names) {
         return sb.toString();
     }
 
+    public static String genStringPlaceholderList(int size) {
```

Comment:
This method likely is not needed https://stackoverflow.com/questions/19033628/how-do-i-pass-a-list-set-array-to-apache-queryrunner-as-parameter-value